packages/core/preboot/core-preboot-server/src/contracts.ts

```diff
@@ -83,3 +50,39 @@ export interface PrebootServicePreboot {
     promise: Promise<{ shouldReloadConfig: boolean } | undefined>
   ) => void;
 }
+
+/**
+ * @public
+ */
+export interface KibanaPrebootServicePreboot
```

Comment:
A public version of `InternalPrebootServicePreboot` to use in mocks, since we cannot use the internal service or internal service types for publically exposed service mocks.